readme.md

```diff
@@ -43,6 +43,26 @@ Force it to be enabled/disabled.
 
 #### .isEnabled(): `Promise<boolean>`
 
+#### .on('change', listener, options): `Function<EventEmitter>`
+
+Attach an event listener that gets called when `.isEnabled()` is changed.
+
+##### options
+
+Type: `Object`
+Default: `{ milliseconds: 3000 }`
```

Comment:
`milliseconds` is not clear. Milliseconds for what? I would name it `pollInterval` and explain what it does.